app/src/productListController.js

```diff
@@ -1,18 +1,17 @@
-/**
- * Created by Szimi7 on 2016.12.07..
- */
 (function () {
     angular
         .module('webshopModule')
         .controller('ProductListCtrl', ProductListController);
 
     function ProductListController($http, $scope) {
-        $http({
-            method: 'GET',
-            url: 'http://localhost:1337/product'
-        }).then(function (result) {
-            $scope.products = result.data;
-        });
+        function getProducts() {
```

Comment:
Having an initializer function in a controller is not a bad idea, but please ensure that it is called when the controller runs.